fml/platform/fuchsia/message_loop_fuchsia.cc

```diff
@@ -10,29 +10,46 @@
 
 namespace fml {
 
-MessageLoopFuchsia::MessageLoopFuchsia()
-    : loop_(&kAsyncLoopConfigAttachToCurrentThread) {}
+MessageLoopFuchsia::MessageLoopFuchsia() : running_(false) {
+  auto status = zx_timer_create(0, ZX_CLOCK_MONOTONIC, &timer_);
```

Comment:
Intuitively, engine subsystems are not resilient to timers firing before their deadlines. `ZX_TIMER_SLACK_LATE` makes more sense and is keeping in line with existing assumptions within the engine. `0 -> ZX_TIMER_SLACK_CENTER ` which may coalesce with earlier timers.